speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/SQLDatabaseStorage.cpp

```diff
@@ -337,15 +333,33 @@ int SQLDatabaseStorage::InsertInitialData(std::ifstream& p_inputFile)
     GET_INT_FROM_RESULTS(environmentId)
 
     // settings
+    // Saved as enum index, but since indices in our code and MySQL are not the same, perform conversion
     std::string interventionMode;
-
     READ_INPUT(p_inputFile, interventionMode);
-
-    values = "'" + interventionMode + "'";
+    switch (std::stoi(interventionMode)) {
+        case INTERVENTION_TYPE_NO_SIGNALS:
+            values = "'Off'";
+            break;
+        case INTERVENTION_TYPE_ONLY_SIGNALS:
+            values = "'Suggest'";
+            break;
+        case INTERVENTION_TYPE_SHARED_CONTROL:
+            values = "'Shared'";
+            break;
+        case INTERVENTION_TYPE_COMPLETE_TAKEOVER:
+            values = "'Force'";
+            break;
+        case INTERVENTION_TYPE_ASK_FOR:
+            std::cout << "'Ask for' intervention type deprecated. Replaced by empty intervention in database" << std::endl;
+            values = "''";
```

Comment:
Do we really need deprecated value handling in unreleased software?

Reply:
I guess not, it just did not seem right to leave one enum value unhandled if it is still present in the code. Additionally, I assumed it would be present in the menu.